src/Command/PluginUnloadCommand.php

```diff
@@ -70,7 +70,7 @@ public function execute(Arguments $args, ConsoleIo $io): ?int
      */
     protected function modifyApplication(string $app, string $plugin): bool
     {
-        $finder = "@\\\$this\-\>addPlugin\(\s*'$plugin'(.|.\n|)+\);+@";
+        $finder = "/\s*\\\$this\-\>addPlugin\(\s*['\"]{$plugin}['\"](\s*,[\s\\n]*\[(\\n.*|.*){0,5}\][\\n\s]*)?\);/m";
```

Comment:
We really should start adding a short comment explaining complex regex.

Reply:
I could use the x modifier and include comments if that would help.

Reply:
That probably isn't needed for this one. I would be happy with a 1-2 line comment stating what it is checking with a quick example. 

Something other than "Checking for correct format with regex" which just re-states the code.